hudi-common/src/main/java/org/apache/hudi/common/table/HoodieTableMetaClient.java

```diff
@@ -581,14 +582,14 @@ public String getCommitActionType() {
         HoodieTableMetaClient
             .scanFiles(getFs(), timelinePath, path -> {
               // Include only the meta files with extensions that needs to be included
-              String extension = HoodieInstant.getTimelineFileExtension(path.getName());
+              String extension = HoodieInstantFormat.getInstantFormat(getTimelineLayoutVersion()).getTimelineFileExtension(path.getName());
```

Comment:
minor:    
   HoodieInstant.getFormat(getTimelineLayoutVersion) seems better than 
   HoodieInstantFormat.getInstantFormat(getTimelineLayoutVersion())

as the format is of the HoodieInstant and HoodieInstantFormat is an internal implementation detail of the HoodieInstant.